framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -1203,14 +1209,22 @@ private void processPostconditionsAndConditionalPostconditions(
                 // report errors here
                 if (e.isFlowParseError()) {
                     Object[] args = new Object[e.args.length + 1];
-                    args[0] =
-                            ElementUtils.getSimpleSignature(
-                                    TreeUtils.elementFromUse(invocationNode.getTree()));
+                    if (n instanceof MethodInvocationNode) {
```

Comment:
Instead of this test, can you just use `elementFromUse` with an `ExpressionTree` and cast to `ExecutableElement`?